views/layout.erb

```diff
@@ -1,17 +1,16 @@
 <!doctype html>
 <html>
   <head>
-    <title>The Landlord</title> 
+    <title>The Landlord</title>
     <link rel="stylesheet" type="text/css" href="/css/styles.css">
   </head>
   <body>
     <nav>
       <a href='/apartments'>List All apartments</a>
-      <a href='/apartments/3'>View sample apartment</a>
+      <a href='/apartments/<%=Apartment.first.id%>'>View sample apartment</a>
```

Comment:
I recommend creating an instance variable in your controller rather than reference an Active Record model from the view. This will keep the view and the engine of this application as separate as possible